sparta/sparta/pipeViewer/Outputter.hpp

```diff
@@ -173,7 +173,7 @@ namespace sparta{
                         // We write the Value for field "i" and only write as much Bytes
                         // as it needs to by checking Sizes[i].
                         writeData_(record_file_,
-                            reinterpret_cast<const char*>(&dat.valueVector[i].first), dat.sizeOfVector[i]);
+                                   reinterpret_cast<const char*>(&dat.valueVector[i].first), dat.sizeOfVector[i]);
```

Comment:
maybe writeData_() could be a templated function to avoid the dreaded reinterpret_cast<>?

Reply:
I normally would agree with this, but this function is used for all transaction types, and we eventually need to turn whatever the data type is into a serialized stream of bytes.  The cast "reminds" us we're doing something potentially dangerous, but I think it's mostly unavoidable unless we went to proper serializers.